taxonomy/algolia/utils.py

```diff
@@ -41,6 +44,83 @@ def index_jobs_data_in_algolia():
     client.replace_all_objects(jobs_data)
     LOGGER.info('[TAXONOMY] Jobs data successfully indexed on algolia.')
 
+def calculate_jaccard_similarity(set_a, set_b):
+    try:
+        intersection = set_a.intersection(set_b)
+        jaccard_similarity = len(intersection)/len(set_a.union(set_b))
+        return jaccard_similarity
+    except:
```

Comment:
this `except` block is too broad, can we narrow it down to more specific exceptions? Also, since we are dividing values here, returning a default float (0.0) would make more sense here instead of int (0).